apps/pie-storybook/stories/testing/pie-chip.test.stories.ts

```diff
@@ -86,13 +86,14 @@ const chipStoryMeta: ChipStoryMeta = {
 
 export default chipStoryMeta;
 
+/* eslint-disable no-console */
```

Comment:
Generally I would prefer disabling eslint inline so that it's easier to remember to remove it when we get around to removing the console statements